app/models/form.rb

```diff
@@ -334,7 +334,11 @@ def sales_or_start_year_after_2022?(type, start_year)
     type == "sales" || (start_year && start_year.to_i > 2022)
   end
 
-  def start_year_after_2024?
+  def start_year_2024_or_later?
     start_date && start_date.year >= 2024
   end
+
+  def start_year_2025_or_later?
+    start_date && start_date.year >= 2025
+  end
```

Comment:
We can use the `start_year_2025_or_later?` method to makes changes conditional for 2025/26.